packages/preset-rules/README.md

```diff
@@ -30,6 +30,8 @@ console.log(a11yResults);
 
 -   Below is the set of rules from [axe rule descriptions](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) and their current mapping to the `base` and `recommended` rulesets in `@sa11y/preset-rules`.
 -   `full` ruleset consists of all rules that are enabled by default in axe.
+-   The default ruleset used by the sa11y APIs is `recommended`.
+    -   This can be overridden using the `SA11Y_RULESET` environment variable.
```

Comment:
I'm a little worried about how much we rely on environment variables. At some point we may consider a separate config file to set all the sa11y specific configs.

Reply:
Yes agree, share the concern as well. 
I have started looking into using separate config (e.g. using https://github.com/indexzero/nconf) - will try to get it done soon in one of the future PRs.